### ISSUE TITLE
Improvements for package build stats GH action

### ISSUE THREAD

**Issue by bryceosterhaus** [liferay/clay#3001]
Currently we build ever package and get its size. This means we include all dependencies and peer dependencies such as `react` and `react-dom`. Below are a few improvements we can make to add more benefit.

- [x] Exclude `react` and `react-dom` from the build sizes. (included in size-limit)
- [x] Add build stats for `clay-css`
    - Add stats for both base and atlas themes
- [x] Only run build stats if packages have changed. (9d5637d27e34f0ecc70517944f5104840ab04e17)
- [x] Calculate the total size if you were to use all packages(this gives us an idea of what the weight on liferay-portal) (`size-limit`)
- [x] Calculate size without `parcel`.(`size-limit`)
- [x] add proper error handling so we dont get green when it actually fails (https://github.com/liferay/clay/pull/3000/checks?check_run_id=503839299)

I am open to more ideas as well, but figured I would start with a handful of items I have been thinking about.

**Comment by jbalsas** [comment]
Just as a reminder of something we did in the past... not saying we should do this, but just in case.

In `2.x` we created a `clay` package that acted as a wrapper of all-things-clay. See https://github.com/liferay/clay/blob/2.x/packages/clay/src/clay.js

This allowed us to recommend a fairly basic initial setup such as:

```html
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/clay/lib/css/atlas.css">
<script src="https://cdn.jsdelivr.net/npm/clay/lib/js/clay.js"></script>
```

Not sure when that file was last updated or if it has everything... but it currently sits at `576Kb` and should contain everything we need to run an app using *anything* from Clay 2.x. 

I feel a similar approach would be a fair initial baseline.
